src/ClassDiscovery.php

```diff
@@ -120,9 +121,9 @@ public static function setStrategies(array $strategies)
     /**
      * Append a strategy at the end of the strategy queue.
      *
-     * @param DiscoveryStrategy $strategy
+     * @param string $strategy FQN to a DiscoveryStrategy
      */
-    public static function appendStrategy(DiscoveryStrategy $strategy)
+    public static function appendStrategy($strategy)
```

Comment:
You are correct. But Im not sure how to write a test for that. `self::$strategies` is private.